tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/experimental/tasks/KotlinNativeCompile.kt

```diff
@@ -53,6 +56,12 @@ open class KotlinNativeCompile @Inject constructor(internal val binary: Abstract
     val libraries: Configuration
         @InputFiles get() = binary.klibraries
 
+    override fun getClasspath(): FileCollection = libraries
+
+    override fun setClasspath(configuration: FileCollection?) {
+        throw UnsupportedOperationException("Use klibraries to set compile classpath in Kotlin/Native")
```

Comment:
klibraries ->klibs

Reply:
`klibraries` here is a name of a task property. But I think it's ok to rename it too.